contracts/interfaces/vaults/IPerpVaultGovernance.sol

```diff
@@ -9,7 +9,14 @@ import "./IPerpFuturesVault.sol";
 import "./IVaultGovernance.sol";
 
 interface IPerpVaultGovernance is IVaultGovernance {
-
+    /// @notice Params that could be changed by Protocol Governance with Protocol Governance delay.
+    /// @param vault Perp Protocol internal vault contract (deposits/withdrawals)
+    /// @param clearingHouse Perp Protocol clearing house contract (open/close positions, add/remove liquidity, liquidate positions)
+    /// @param accountBalance Perp Protocol account balance contract (get position total value, add/remove base token)
+    /// @param vusdcAddress Reference to Perp Protocol vUSDC (virtual USDC, it`s amount is equal to USDC amount multiplied by the current leverageMultiplier)
```

Comment:
'it`s amount is equal to USDC amount multiplied by the current leverageMultiplier' this is not true. Just write what is vUSDC and this is enough

Reply:
okay, I got it